libs/service/service.go

```diff
@@ -25,19 +25,7 @@ type Service interface {
 	// Start the service.
 	// If it's already started or stopped, will return an error.
 	// If OnStart() returns an error, it's returned by Start()
```

Comment:
I think we should update this to document how the context is used:

```suggestion
	// Start is called to start the service, which should run until
	// the context terminates. If the service is already running, Start
	// must report an error.
```

I'm actually not sure how important the latter part is, but it is consistent with the existing behaviour (I don't know if anything cares, though).

I left out mention of `OnStart` since this interface no longer carries that method. But see also my comment further down.